src/meta/src/hummock/manager/tests.rs

```diff
@@ -207,20 +215,22 @@ async fn test_hummock_compaction_task() {
         StaticCompactionGroupId::StateDefault.into(),
     )
     .await;
-    commit_from_meta_node(
```

Comment:
Thanks for cleaning this up. I just realize that there are many variants of commit epoch calls in UT and their implementations are using magic numbers.